sidebars.ts

```diff
@@ -84,7 +84,6 @@ module.exports = {
           collapsed: true,
           link: {type: 'doc', id: 'send-data/opentelemetry-collector/index'},
           items: [
-            'send-data/opentelemetry-collector/quickstart',
```

Comment:
Why is the quickstart being deleted @JV0812 ?

Reply:
@kimsauce The content in the quickstart does not go good with the current UI and the work is going on to update the UI accordingly. Also, I stashed this line to hide the doc from the side, but as per your comment I just deleted the path.

Reply:
Oh I see what you're saying. In this case it's ok to hide it like you did before, and maybe add a note (commented out) at the top of the quickstart like this:

```
---
id: quickstart
title: OpenTelemetry Collector Quickstart
sidebar_label: Quickstart
description: Get up and running quickly with the Sumo Logic OpenTelemetry Collector.
---

<!-- temporarily pulled from docs; UI screenshots being updated -->

import useBaseUrl from '@docusaurus/useBaseUrl';

Sumo Logic makes it easy to get started with OpenTelemetry by providing a custom distribution of the OpenTelemetry collector, which can be used to send metrics, logs, and traces directly to our platform.
```